includes/validation/class-amp-validation-manager.php

```diff
@@ -1946,15 +1927,15 @@ public static function unserialize_validation_error_messages( $serialized ) {
 	public static function get_validate_url_error_message( $error_code, $error_message = '' ) {
 		$check_error_log = sprintf(
 			/* translators: %s is link to Debugging in WordPress */
-			' ' . __( 'Please check your server PHP error logs; to do this you may need to <a href="%s" target="_blank">enable</a> <code>WP_DEBUG_LOG</code>.', 'amp' ),
+			__( 'Please check your server PHP error logs; to do this you may need to <a href="%s" target="_blank">enable</a> <code>WP_DEBUG_LOG</code>.', 'amp' ),
```

Comment:
Translations should be escaped, as the actual HTML that will be sent to the browser is crowd-sourced user-input.

Reply:
True. I was not doing it here because the function notes that it returns “Error message with HTML markup.” When the return value is used, it is getting passed through `wp_kses()` when the error is actually being printed:

https://github.com/ampproject/amp-wp/blob/2a1dc540e6ecc2621a48cf4324308b2c23f7b6ce/includes/validation/class-amp-validated-url-post-type.php#L1281-L1291

But I suppose it would be a good idea to add more hardening.

Reply:
See d48ceae6f50623a6e0abb63057f7dd294e98d54c